tests/test_models.py

```diff
@@ -431,6 +466,7 @@ class UserAdmin(ModelView, model=User):
     assert UserAdmin().get_export_columns() == [
         ("addresses", User.addresses.prop),
         ("profile", User.profile.prop),
+        ("permissions", User.permissions.prop),
```

Comment:
Do any of these tests actually test this behaviour?

Reply:
perhaps some tests for url generations need to be added

Reply:
ok, i removed these tests